diff -urN -x '*.orig' -x .version conda-package-handling-1.7.3.orig/setup.py conda-package-handling-1.7.3.ptch/setup.py
--- conda-package-handling-1.7.3.orig/setup.py	2021-04-12 17:49:45.000000000 +0800
+++ conda-package-handling-1.7.3.ptch/setup.py	2022-02-24 02:14:57.947000000 +0800
@@ -5,7 +5,7 @@
 from setuptools.extension import Extension
 from Cython.Build import cythonize
 
-_libraries = ["archive_and_deps"]
+_libraries = ["archive"]
 if sys.platform == 'win32':
     _libraries.append('advapi32')
     _libraries.append('user32')
@@ -30,7 +30,7 @@
     author="Anaconda, Inc.",
     author_email='conda@anaconda.com',
     url='https://github.com/conda/conda-package-handling',
-    ext_modules=cythonize([archive_utils_cy_extension]),
+    ext_modules=cythonize([archive_utils_cy_extension], language_level = "3"),
     packages=find_packages("src", exclude=["tests"]),
     package_dir={"": "src"},
     entry_points={
