--- conda-package-handling-1.8.1.orig/setup.py	2022-04-02 00:26:46.000000000 +0800
+++ conda-package-handling-1.8.1.ptch/setup.py	2022-04-22 21:18:21.289000000 +0800
@@ -5,7 +5,7 @@
 from setuptools.extension import Extension
 from Cython.Build import cythonize
 
-_libraries = ["archive_and_deps"]
+_libraries = ["archive"]
 if sys.platform == 'win32':
     _libraries.append('advapi32')
     _libraries.append('user32')
@@ -24,7 +24,7 @@
     author="Anaconda, Inc.",
     author_email='conda@anaconda.com',
     url='https://github.com/conda/conda-package-handling',
-    ext_modules=cythonize([archive_utils_cy_extension]),
+    ext_modules=cythonize([archive_utils_cy_extension], language_level = "3"),
     packages=find_packages("src", exclude=["tests"]),
     package_dir={"": "src"},
     entry_points={
